src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.spring.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+    @Override
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+        auth.inMemoryAuthentication()
+                .withUser("bob@i.ua")
+                .password(getEncoder().encode("123456"))
+                .roles("USER");
+    }
+
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .authorizeRequests()

```

Comment:
imho, that looks pretty strange, optional:
```suggestion
        http.authorizeRequests()
```

Reply:
such example was in lesson

Reply:
I know) that's why I wrote "imho" 🤓